src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -1028,13 +1044,15 @@ int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
     if (!resolve_sdk_dotnet_path(own_dir, &sdk_dotnet))
     {
         assert(argc > 1);
-        if (pal::strcasecmp(_X("--help"), argv[1]) == 0 ||
-            pal::strcasecmp(_X("--version"), argv[1]) == 0 ||
-            pal::strcasecmp(_X("-h"), argv[1]) == 0 ||
-            pal::strcasecmp(_X("-v"), argv[1]) == 0)
+        if ( pal::strcasecmp(_X("-h"), argv[1]) == 0 ||
+             pal::strcasecmp(_X("-h"), argv[1]) == 0 )
         {
             return muxer_usage(true);
         }
+        else if (pal::strcasecmp(_X("--version"), argv[1]) == 0)
+        {
+            return muxer_info();
```

Comment:
muxer_info should be a void function since it is doing what is expected of it. Also, the return value from here should be a normal exit since the host did what was requested of it.